src/components/dataTable/AlleleCell.js

```diff
@@ -4,7 +4,7 @@ import {Link} from 'react-router-dom';
 import AlleleSymbol from '../../containers/allelePage/AlleleSymbol';
 
 const AlleleCell = ({allele}) => {
-  return (
+  return allele.category === 'variant' ? allele.symbol : (
```

Comment:
I am not sure this is the right logic. Is this the first column? It should be allele.symbol in all cases. It will be populated by the API (even though, allele.symbol would be a bit confusing) transparently.

Reply:
We don't have allele pages for HTP variants. It will link to a non-existent page that shows 404 if keep the link.

Reply:
Oh, now I see the logic. Looks good.